src/boot/stylesheets/main.scss

```diff
@@ -165,22 +165,17 @@ body {
     	background-color: $white;
     	color: $gray-text-min;
     	border-bottom: 1px solid lighten( $gray, 30 );
+        border-left: 1px solid $white;
```

Comment:
I think all of `main.scss` needs a reformat... or my editor config is messed up?